compiler/src/iree/compiler/InputConversion/MHLO/MHLOToMHLOPreprocessing.cpp

```diff
@@ -694,13 +789,13 @@ struct ScatterMaterializeInsertedDim
     auto operand = op.getInputs().front();
     auto indicesTy = indices.getType().cast<ShapedType>();
     auto operandTy = operand.getType().cast<ShapedType>();
+    auto dimNumbers = op.getScatterDimensionNumbers();
```

Comment:
Why is this change needed?

Reply:
It is not required so I reverted this part of the change.